apps/pie-docs/src/components/tag/overview/sizes.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Where is this file used?

Reply:
it's used on overview.md

Reply:
it isn't used anymore you are right. removed